src/main/java/com/shevtsov/utilities/MyUtilities.java

```diff
@@ -0,0 +1,23 @@
+package com.shevtsov.utilities;
+
+import java.util.Scanner;
+
+public class MyUtilities {
```

Comment:
try to avoid personal class

Reply:
change to 'viewUtilities' becouse utilities for view-classes are there.